index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="styles.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Survey Page</title>
+</head>
+<body>
+    <h1 id="title">Microverse Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+    <form action="#" id="survey-form">
+        <label for="name" id="name-label">Write your name</label><br>
+        <input type="text" id="name" placeholder="Enter your name"><br>
+        <label for="email" id="email-label">Write your email</label><br>
+        <input type="email" id="email" value="email" placeholder="Enter your Email"><br>
+        <label for="number" id="number-label">Write your number</label><br>
+        <input type="number" id="number" min="0" max="9" placeholder="Enter a number"><br>
```

Comment:
Please use class for elements within the same family instead of id. It will make your code dry while styling it.